crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -1644,13 +1714,15 @@ fn handle_exports_imports_field(
     path: &str,
     conditions: &BTreeMap<String, ConditionValue>,
     unspecified_conditions: &ConditionValue,
+    query: Vc<QueryMap>,
 ) -> Result<Vc<ResolveResult>> {
     let mut results = Vec::new();
     let mut conditions_state = HashMap::new();
     let values = exports_imports_field
         .lookup(path)
```

Comment:
```suggestion
        .lookup(path + query)
```

query is part of the lookup

Reply:
Doesn't seem right to me, but sure

wouldn't be able to use `?module` for wasm module behind an exports field